automated install/basic-install.sh

```diff
@@ -1787,21 +1787,12 @@ displayFinalMessage() {
 \\n${additional}" "${r}" "${c}"
 }
 
-update_dialogs() {
-    # If pihole -r "reconfigure" option was selected,
-    if [[ "${reconfigure}" = true ]]; then
-        # set some variables that will be used
-        opt1a="Repair"
-        opt1b="This will retain existing settings"
-        strAdd="You will remain on the same version"
-    else
-        # Otherwise, set some variables with different values
-        opt1a="Update"
-        opt1b="This will retain existing settings."
-        strAdd="You will be updated to the latest version."
-    fi
-    opt2a="Reconfigure"
-    opt2b="Resets Pi-hole and allows re-selecting settings."
+repair_dialog() {
```

Comment:
Correct me if I'm wrong, but does this now mean that `pihole -r` will still display a dialog box, but the dialog box will only have one "option" to select?

If so, does that maybe render the dialog box superfluous?

Reply:
I just left it to give users the chance to cancel the process. If you think it's not necessary, I can also just remove it.

Reply:
I don't think repairing is a destructive process that needs to have a cancel option. My vote would be to remove the dialog.

Reply:
Removed it.